index.rst

```diff
@@ -44,14 +44,648 @@
+
+The Watcher itself is not a display tool nor does it have any display functionality.
+When a condition defined by an rule is met, an alarm is generated and the observer is alerted via a LOVE screen.
+The alarm has a series of levels and audible alerts are sent out via LOVE, as well as a visual notification.
+Once the alarm is acknowledged by the observer the alert is considered to be completed.
+There is no feedback or interaction for the observer beyond the acknowledgment of the alarm.
+Alarms will only deactivate (stop re-occuring) if the condition for which they are activated cease to exist, e.g., if a CSC resumes publishing heartbeats after being restored.
+
+
+SAL Scripts
+'''''''''''
+
+`SAL scripts <https://ts-salobj.lsst.io/sal_scripts.html#lsst-ts-salobj-sal-scripts>`_ are a series of coordinated sequences, often consisting of commands to CSCs, that are executed by the `ScriptQueue <https://ts-scriptqueue.lsst.io/>`_.
+The LOVE system contains a graphical ScriptQueue interface to control and monitor the execution of scripts.
+It is anticipated that most standard operations will utilize scripts.
+Also possible, although not standard practice, is to manually execute a script from the `Nublado (Jupyter Notebook) Interface`_.
+From within a SAL script, users can send standard commands to components as well as send data to the `OCPS`_ for processing.
+The script can then either wait for the analysis to complete and continue, with the ability to act based on the result, or launch the process (e.g. image reduction) and continue executing the script.
+Scripts are not intended to perform intensive data analysis and, in general, are not expected to produce artifacts.
+Scripts are also unable to display visualizations.
+
+
+OCPS
+''''
+The `Observatory Controlled Pipeline Service (OCPS) <https://dmtn-133.lsst.io/>`_ is a CSC that allows observers (and SAL Scripts) to execute pipeTasks to perform data reductions and analyses.
+The CSC runs on the summit but the data processing is currently running at the base on the commissioning cluster (although it may be relocated to the summit).
+The OCPS is not a display tool, but can be used to produce artifacts (such as images, spectra etc) that observers want to display.
+The current scope of this service is to only provide image-related processing.
+It cannot currently query the EFD.
+
+At this time, the OCPS is being used to perform the analysis of daily calibrations executed from the scriptQueue.
+
+
+Prompt Processing
+'''''''''''''''''
+The Prompt Processing Pipeline is expected to run at the United States Data Facility (USDF).
+Within 60s, the images taken on-sky get reduced and a series of data products are made available.
+A small number of these data products are sent back to the summit `via the Telemetry Gateway <ls.st/lse-72>`_.
+This service is not yet in place.
+
+It is expected that metrics coming from Prompt Processing (and `faro`_) will be used in on-the-fly displays.
+
+faro
```

Comment:
Just a general comment.. it seems like everywhere "faro" is referred to with lower-case, even at the start of a phrase or paragraph..  Is it an acronym? If so, should it be "FARO" instead?

Reply:
it's not an acronym.... I don't actually know why it's called "faro"

Reply:
faro is Spanish for lighthouse. It's a reference to La Serena and also the idea of helping to light the way / navigate